fe/fe-core/src/main/java/com/starrocks/analysis/ArithmeticExpr.java

```diff
@@ -425,6 +425,60 @@ public static Type findCommonType(Type t1, Type t2) {
         }
     }
 
+    public static Type getCommonType(Type t1, Type t2) {
```

Comment:
Can be replace all `ArithmeticExpr.findCommonType`?

Reply:
I don't know what the side effects will be, I think it should be replaced in another pr.

Reply:
> Can be replace all `ArithmeticExpr.findCommonType`?

@Seaven This function is still used by old planner, we can delete it later

Reply:
> > Can be replace all `ArithmeticExpr.findCommonType`?
> 
> @Seaven This function is still used by old planner, we can delete it later

OK, record issue to remove it later plz